README.md

```diff
@@ -139,17 +139,21 @@ You'll need a C3 or C3X and a Toyota A harness for current supported vehicles.
 
 ### Step 1. Power up C3X, connect to Wi-Fi, and install `commaai/master-ci`
```

Comment:
I wonder if we should just refer everything as a C3 and just make a note here that this applies to the newer C3X as well. Sometimes we do get people asking if a C3 would work.

Reply:
That could work. I was thinking that the newbies who need this have C3X

Reply:
Mind if I do this on a different PR? I think we should get the DisableUpdates change pushed out soon